src/dev_entity_worker.py

```diff
@@ -544,14 +543,17 @@ def get_expanded_dataset_metadata(self, dataset_id:Annotated[str, 32], valid_use
                                      , user_info:dict) -> dict:
         # Retrieve the metadata dictionary for the Dataset, which will be expanded later to hold entries for the
         # associated data.
-        expanded_dataset_dict = self._get_entity_by_id_for_auth_level(entity_id=dataset_id
-                                                                     , valid_user_token=valid_user_token
-                                                                     , user_info=user_info)
+        dataset_dict = self._query_target_entity(entity_id=dataset_id)
+
         # Confirm the dataset_id passed in is for a Dataset entity.
-        if expanded_dataset_dict['entity_type'] not in ['Dataset','Publication']:
-            raise entityEx.EntityBadRequestException(   f"Only Dataset provenance metadata can be retrieved."
-                                                        f" An entity of type '{expanded_dataset_dict['entity_type']}'"
-                                                        f" for uuid={expanded_dataset_dict['uuid']} is not supported.")
+        if dataset_dict['entity_type'] not in ['Dataset', 'Publication']:
```

Comment:
Switch to use `schema_manager.entity_type_instanceof()` to be consistent with other places. And if later we'll add a new subclass of Dataset, we won't have to come back.